coldfront/core/project/templates/project/project_join_list.html

```diff
@@ -9,6 +9,46 @@
 
 {% block content %}
 <h1>Join a Project</h1><hr>
+<h5>Pending Join Requests</h5>
+  {% if user.userprofile.access_agreement_signed_date is None %}
+    <p>First Review and Sign the cluster user agreement. Only then you can Join a cluster project and gain access to the cluster.</p>
+  {% elif join_requests is None %}
```

Comment:
This isn't showing up for me. A query with no results would give an empty queryset, which isn't equal to `None`.

`{% elif not join_requests %}` should work.